spec/support/shared_examples/create_service_with_price_rule_shared_examples.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.shared_examples "creates a product with billing mode" do |product_type, billing_mode|
+  let(:logged_in_user) { administrator }
+
+  it "can create a #{product_type}, which automatically creates a price rule" do
+    visit facility_products_path(facility)
+    all("a", text: product_type.capitalize)[0].click
```

Comment:
Why do we need to get `all` and then select the first one?

Reply:
The reason behind this is that when the product type is service, there's another link in the menu called "Timed Services" that interferes when calling `click_link "Services"`. With this code, I find all the links that contain the word "Services" (there are only 2) and select the first one, which is the link I need to visit.

Reply:
Oh, I think that you can use something like `click_link("Services", href: facility_services_path(facility))`